cypress/e2e/bulk-edit/permissions/bulk-edit-csv-query-permissions.cy.js

```diff
@@ -0,0 +1,42 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import testTypes from '../../../support/dictionary/testTypes';
+import permissions from '../../../support/dictionary/permissions';
+import devTeams from '../../../support/dictionary/devTeams';
+import users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+
+let user;
+
+describe('bulk-edit', () => {
+  describe('permissions', () => {
+    before('create test users', () => {
+      cy.createTempUser([
+        permissions.bulkEditCsvView.gui,
+        permissions.bulkEditCsvEdit.gui,
+        permissions.bulkEditUpdateRecords.gui,
+        permissions.bulkEditQueryView.gui,
+      ])
+        .then(userProperties => {
+          user = userProperties;
+          cy.login(user.username, user.password, { path: TopMenu.bulkEditPath, waiter: BulkEditSearchPane.waitLoading });
+        });
+    });
+
+    after('delete test data', () => {
+      users.deleteViaApi(user.userId);
```

Comment:
Please use a capital letter for an import.

Reply:
fixed